src/evict/evict_page.c

```diff
@@ -233,7 +233,7 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
         if (!closing)
             __evict_exclusive_clear(session, ref, previous_state);
 
-        if (LF_ISSET(WT_EVICT_CALL_URGENT)) {
+        if (time_start != 0 /* WT_EVICT_CALL_URGENT */) {
```

Comment:
Same here.

Reply:
Answered in the previous, related comment.